src/mesh/python/x3d_generator.py

```diff
@@ -117,13 +124,14 @@
 for cell in range(mesh.num_cells):
     for j in range(mesh.num_faces_per_cell[cell]):
         # -- get the full face index for the cell and local face index
-        face = mesh.faces_per_cell[cell, j]
+        #face = mesh.faces_per_cell[cell, j]
+        face = mesh.faces_per_cell[cell][j]
         # -- get the number of nodes for this face
         nnpf = mesh.num_nodes_per_face[face]
         # -- build face string
         face_str = '{0:10d}{1:10d}'.format(face + 1, nnpf)
         # -- append node indices to face string
-        for node in mesh.nodes_per_face[face, :]:
+        for node in mesh.nodes_per_face[face]:#, :]:
```

Comment:
Is this in optimization?

Reply:
This syntax just works for numpy arrays and lists of numpy arrays. I now realize that I could technically still be using a numpy array for my `nodes_per_face`, but in 3D I wouldn't be able to do that so I figured this could just stay in

Reply:
Right, that makes sense.